src/app/routers/user.ts

```diff
@@ -19,19 +21,26 @@ const BASE_ROUTE_PATH = "/users";
 
 const userRouter = Router()
   // get a list
-  .get("list", validateParamsForSuggestions, userController.getSuggestions)
+  .get(
+    "list",
+    checkToken,
+    validateParamsForSuggestions,
+    userController.getSuggestions
+  )
   // get a user
-  .get("/:id", validateIdInParams, userController.getOneById)
+  .get("/:id", checkToken, validateIdInParams, userController.getOneById)
   // create a user
-  .post("/", validateCreateUserBody, userController.createOne)
+  .post("/", checkToken, validateCreateUserBody, userController.createOne)
```

Comment:
Is it really necessary to check the token on user creation? It seems to me, this is a mistake. WDYT?